samples/react-native/.detoxrc.js

```diff
@@ -44,13 +44,26 @@ module.exports = {
         },
       },
     },
-    'ci.sim': {
+    'ci.sim.auto': {
```

Comment:
This should fix the [CI failure](https://github.com/getsentry/sentry-react-native/actions/runs/13518574157/job/37774448447?pr=4588) on https://github.com/getsentry/sentry-react-native/pull/4588 after https://github.com/getsentry/sentry-react-native/pull/4588/commits/d3369032eccccf50c3c24a8d0628a23580bc7bcc :)

Reply:
Good catch, thank you, it should be fixed after https://github.com/getsentry/sentry-react-native/pull/4593/commits/03cd6bafb1ab6716611fa9001acc3d612cd2d74a